ui/app/src/components/MarketplaceDialog/InstallPluginDialogContainer.tsx

```diff
@@ -214,7 +214,8 @@ export function InstallPluginDialogContainer(props: InstallPluginDialogProps) {
             styles={{
               root: {
                 flexGrow: 1,
-                justifyContent: 'center'
+                justifyContent: 'center',
+                minHeight: 'unset'
```

Comment:
What happens without this?

Reply:
The LoadingState default minHeight is '100%', and in this view it causes a scroll bar